06-lifecycle_of_a_Patch.Rmd

```diff
@@ -22,6 +22,71 @@ To submit a patch, you need:
 
 Refer to the guidelines given [here](https://www.r-project.org/bugs.html#how-to-submit-patches) for the process to prepare a patch against R-devel.
 
+You should modify the source code on your computer of the latest version of R (`r-devel`). You can retrieve the latest source code of R via:
+
+  ```sh
+  export TOP_SRCDIR="$HOME/Downloads/R"
+  svn checkout https://svn.r-project.org/R/trunk/ "$TOP_SRCDIR"
+  ```
+Then you should edit the files, for example a documentation file such as  `"$TOP_SRCDIR"/src/library/stats/man/Multinom.Rd`. 
+Make your desired changes to that or more files.
+
+Then you should check that R still works as expected via:
+
+  ```sh
+  cd "$TOP_SRCDIR"
+  make check-devel
+  ```
+
+If there is no test for your proposed change you can add a new test.
+
+Then you should bring changes from the repository into the working copy, in case any other change has been introduced, and create a path.diff file with just the changes you want to propose to the R core:
+
+  ```sh
+  svn update
+  svn diff > patch.diff
+  ```
+  
+This `patch.diff` file is the one that can be proposed to the R core via [Bugzilla](#SubmitPatches). You can also [ask for reviews](#PatchesReview) to the patch before proposing it to the R core via the [r-devel mailing list](https://stat.ethz.ch/mailman/listinfo/r-devel) or the slack channel of the R-contributors space.
+
+Besides checking in your computer, you can use the Github mirror  [r-devel/r-svn](https://github.com/r-devel/r-svn  "A github svn mirror") of the source code to check this patch with different configurations and OS.
```

Comment:
I think it would be helpful to create a sub-section for the git workflow, so that we can point to it as a quick way (but not the recommended approach) for people to make a change, test it, and create a patch, without needing to have set up to build R from source.